src/PerfViewExtensions/GlobalSrc/Global.csproj

```diff
@@ -11,24 +11,6 @@
     <AssemblyName>Global</AssemblyName>
     <TargetFrameworkVersion>v4.5</TargetFrameworkVersion>
     <FileAlignment>512</FileAlignment>
-    <!-- Determine if we are building 'next to' the main PerfView sources -->
```

Comment:
Thank you for this explanation. I believe we could improve this scenario in the future in several ways, but totally agree that it would best be done as a separate and very intentional change. The original behavior has been restored for the extensibility case while preserving the new behavior (project references) for building **PerfView.sln**.